src/main/java/net/smartcosmos/edge/things/rest/template/thing/ThingRestConnector.java

```diff
@@ -3,153 +3,71 @@
 import lombok.extern.slf4j.Slf4j;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpMethod;
 import org.springframework.http.RequestEntity;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 
-import net.smartcosmos.edge.things.config.SmartCosmosEdgeThingsProperties;
 import net.smartcosmos.edge.things.domain.local.things.RestPagedThingResponse;
 import net.smartcosmos.edge.things.domain.local.things.RestThingCreate;
 import net.smartcosmos.edge.things.domain.local.things.RestThingCreateResponseDto;
 import net.smartcosmos.edge.things.domain.local.things.RestThingResponse;
 import net.smartcosmos.edge.things.domain.local.things.RestThingUpdate;
 import net.smartcosmos.edge.things.rest.RestTemplateFactory;
 import net.smartcosmos.edge.things.rest.template.SmartCosmosRequest;
+import net.smartcosmos.edge.things.rest.request.ThingRequestFactory;
 
 @Component
 @Slf4j
+@Deprecated
```

Comment:
Deprecation of internally used class:
Please add JIRA ticket to update usages of this class to epic https://smartractechnology.atlassian.net/browse/OBJECTS-851